src/modules/fastify/index.ts

```diff
@@ -14,12 +17,42 @@ export async function setupFastify(): Promise<FastifyInstance> {
   });
   let exportedApp = null;
 
+  app.setErrorHandler((err, req, reply) => {
+    if (err.validation) {
+      reply.status(500).send({
+        status: 500,
+        error: err.message,
+      });
+      return;
+    }
+
+    if (err instanceof StatusError) {
+      reply.status(err.statusCode).send({
+        status: err.statusCode,
+        error: err.message,
+      });
+      return;
+    }
+
+    log.error('unhandled exception on server:', err);
+    reply.status(500).send({
+      status: 500,
+      error: 'Internal server error',
+    });
+  });
+
   // plugins & routes
   log.info(`setting up plugins and routes`, { evt: 'setup-plugins' });
+  await app.register(cors, {
+    origin: config.server.cors.split(' ').filter((v) => v.length),
```

Comment:
```suggestion
    origin: config.server.cors.split(' ').filter((v) => v.length > 0),
```

I know this works with falsy, but it would probably be better for code clarity to have it explicit